src/components/App/App.js

```diff
@@ -26,6 +25,7 @@ import {
   UserRecipes,
   UserSettings
 } from 'pages';
+import LazyLoaderFallback from 'components/LazyLoaderFallback/LazyLoaderFallback';
```

Comment:
Does the name `SuspenseFallback` sound OK to you? It's a bit shorter and more obvious from the React API angle.

Reply:
Sounds good to me, I'll make those changes.